### PR TITLE
#702 Geometry Collection now supports serde.

### PR DESCRIPTION
**Pull request by martinfrances107**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).

Initial version ... I want to discuss ways to test this, from what I can see serde support is not tested for geometry primitives like point, or line_string.. so I don't know how to confirm this works for all users.

### CODE REVIEW

**Comment by michaelkirk** [comment]
I we'd also want to derive serde for `Geometry` - can you add that here as well?

**Comment by martinfrances107** [comment]
> I think we'd also want to derive serde for Geometry - can you add that here as well?

That seems like a good idea... done.

**Comment by michaelkirk** [comment]
bors try

**Comment by michaelkirk** [comment]
LGTM

Personally, I don't think we need additional tests for this particular change, but I'll wait a couple days before merging in case someone else wants to weigh in.

**Comment by lnicola** [comment]
Needs a changelog entry.

**Comment by martinfrances107** [comment]
> I don't think it needs tests, but it's worth adding a changelog entry.

Thanks .. I agree... done.

File: geo-types/CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 * Support `rstar` version `0.9` in feature `use-rstar_0_9`
   * <https://github.com/georust/geo/pull/682>
+* Geometry and GeometryCollection now support serde.
```
Comment (lnicola) [inline geo-types/CHANGES.md:7]:
```suggestion
* `Geometry` and `GeometryCollection` now support serde.
```

**Comment by martinfrances107** [comment]
@lnicola  That is better .. thank you.

**Pull request merged**